### DISCUSSION TITLE
How to track fungible asset creation given `primary_fungible_store::create_primary_store_enabled_fungible_asset` is not emitting event

### DISCUSSION THREAD

**Discussion by 0xbe1** [aptos-labs/aptos-developer-discussions#418] [Questions] [answered]
### Describe your question in detail.

See title. I would like to track what are the fungible assets being created but cannot find a event to track on.

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

N/A

**Comment by ying-w** [comment, marked as answer]
You'll want to filter on changes to supply resource
